src/session/error.rs

```diff
@@ -91,6 +91,9 @@ pub enum ParseError {
     Malformed(Box<dyn std::error::Error + Send + Sync>),
     /// An error occurred when decoding bytes of a proto message.
     Decode(prost::DecodeError),
+    /// An error occurred when converting time micros from proto
+    /// to `std::time::SystemTime`.
+    TimeMicrosConversion(TimeMicrosConversionError)
```

Comment:
Likewise, I don't think we should add too much variants into the `ParseError`. `TmeMicrosConversion` falls under `Malformed` IMO.

Reply:
Removed the variant - merged it into `Malformed`.